core-build/gulp-core-build/src/tasks/jest.schema.json

```diff
@@ -13,6 +13,14 @@
       "type": "boolean",
       "default": false
     },
+    "cache": {
```

Comment:
>cache [](start = 5, length = 5)

cacheEnabled ? this would be similar to isEnabled

Reply:
Trying to be consistent with jest config.

---
In reply to: [153653897](https://github.com/Microsoft/web-build-tools/pull/453#discussion_r153653897) [](ancestors = 153653897)